packages/cactus-test-tooling/src/main/typescript/public-api.ts

```diff
@@ -13,6 +13,17 @@ export {
   IBesuMpTestLedgerOptions,
 } from "./besu/besu-mp-test-ledger";
 
+export {
+  DamlTestLedger
+  // IDamlTestLedgerConstructorOptions,
+  // DAML_TEST_LEDGER_DEFAULT_OPTIONS,
+  // DAML_TEST_LEDGER_OPTIONS_JOI_SCHEMA,
+} from "./daml/daml-test-ledger";
+
+// export {
+//   BesuMpTestLedger,
```

Comment:
is this a typo/mistake? Why are we commenting the exporting of the besu test ledger?

Reply:
Hello @RafaelAPB ,
Yes this is just a typo mistake. I wasn't able to switch this to draft PR before I pushed this :)

Cheers!
Rayn